honeybee/_hbanalysissurface.py

```diff
@@ -615,12 +628,12 @@ def to_json(self):
         """Get HBSurface as a dictionary.
             {"name": "",
             "vertices": [[(x, y, z), (x1, y1, z1), (x2, y2, z2)]],
-            "surface_type": null  // 0: wall, 5: window
+            "surface_material": {}  // radiance material json file
             }
         """
         return {"name": self.name,
                 "vertices": [[tuple(pt) for pt in ptgroup] for ptgroup in self.points],
-                "surface_type": self.surface_type.type_id
```

Comment:
Same as what I said above. We need to keep `surface_type` in the json schema for HBSurface.